### ISSUE TITLE
Supporting "gray" start instead of "outlined" star for empty star

### ISSUE THREAD

**Issue by zirho** [bviebahn/react-native-star-rating-widget#26]
Hi, 
I am enjoying using the library a lot.
Is there a way to replace a shape of empty star?

![image](https://user-images.githubusercontent.com/1251296/202824846-7c486744-284f-4342-afa0-f196125c25c7.png)

Or I could try to make PR for that. 

Thanks!

**Comment by bviebahn** [comment]
Hi,

yes I think that would be useful. I would suggest adding a `StarComponent` prop with a type something like this:
```typescript
StarComponent?: (props: { type: 'full' | 'half' | 'empty'; size: number; color: string }) => React.ReactNode
```
What do you think?

**Comment by bviebahn** [comment]
Alright, let me know if you would lilke to work on it. Otherwise I can probably find some time tomorrow.

**Comment by zirho** [comment]
Sorry I am spending too much time debugging some ambiguous RN issue.
I will come back once I get it fixed. 
It would be awesome if you can get to it. 

Gnite!

**Comment by bviebahn** [comment]
Ok sure, good luck with that!

**Comment by bviebahn** [comment]
I just published a new version 1.4.1 which includes this change. Let me know if something doesn't work as expected.
